camel/agents/chat_agent.py

```diff
@@ -420,12 +420,7 @@ def step(
                 a boolean indicating whether the chat session has terminated,
                 and information about the chat session.
         """
-        if (
-            isinstance(self.model_type, ModelType)
-            and "lama" in self.model_type.value
-            or isinstance(self.model_type, str)
-            and "lama" in self.model_type
-        ):
+        if "lama" in self.model_type.value:
```

Comment:
Thanks @lightaime , I used 'lama' instead of 'llama' since for some platform the model name could be 'Llama', yes I will expand this feature to all open source models for issue: https://github.com/camel-ai/camel/issues/970 in this week

Reply:
@WHALEEYE maybe we need to set one more property like 'support_native_tool_calling' for models from openai, mistral etc..